tests/test_icml_conference.py

```diff
@@ -1343,6 +1343,15 @@ def test_forum_chat(self, openreview_client, helpers):
                         "id": "all",
                         "label": "All"
                     },
+                    {
+                        "id": "discussion",
+                        "label": "Discussion",
+                        "filter": "-invitation:ICML.cc/2023/Conference/Submission${note.number}/-/Chat",
```

Comment:
I wanted to add this because I think it will be a common setup - to have 1 tab for the chat messages and one standard layout tab with other types of messages.